tests/foreman/ui/test_subscription.py

```diff
@@ -555,3 +560,80 @@ def test_positive_prepare_for_sca_only_subscription(target_sat, function_entitle
             'This organization is not using Simple Content Access. Entitlement-based subscription '
             'management is deprecated and will be removed in Satellite 6.16.' in sca_alert
         )
+
+
+@pytest.mark.parametrize('setting_update', ['expire_soon_days'], indirect=True)
+def test_positive_check_manifest_validity_notification(
+    target_sat, setting_update, function_org, function_sca_manifest
+):
+    """Check notification when manifest is going to expire.
+
+    :id: 29ab0e80-25eb-44f0-9294-3a26922c33c5
+
+    :setup:
+        1. create new organization
+
+    :steps:
+        1. Upload expired manifest in newly created org
+        2. Go to Content > Subscriptions page, click on 'Manage Manifest' button.
+        3. Search for message string 'Manifest expired', or 'Your manifest expired'
+        4. Delete expired manifest from this org (cleanup part)
+        5. Upload non-expired manifest
+        6. Go to Content > Subscription page, click on 'Manage Manifest' button.
+        7. Search for message string 'Manifest expiring soon', or 'Your manifest will expire'
+        8. Delete expired manifest from this org (cleanup part)
+
+    :expectedresults:
+        1. 'Manifest expired', 'Manifest expiring soon' messages appear on Manage Manifest modal box
+
+    :BZ: 2075163
+
```

Comment:
Now we can use the brand new docstring keywords
```suggestion

    :Verifies: SAT-11630
    
```